app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LikesHelper
+end
```

Comment:
- [ ] Kindly remove this generated file.

Reply:
> * [ ]  Kindly remove this generated file.

- [ ] Kindly remove this as requested by the previous reviewer.